### PR TITLE
Docs version 3

### PR DESCRIPTION
**Pull request by antoniomuso**
## 🚨 Proposed changes
Documentation version v3

## ⚙️ Types of changes

What types of changes does your code introduce? _Put an `x` in the boxes that apply_

-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] Documentation Update (if none of the other choices apply)
-   [ ] Refactor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,42 +113,82 @@ const overviewContent = (
...
 ```yaml
 integrations:
     gitlab:
         - host: gitlab.com
           token: ${GITLAB_TOKEN}
 ```
+**Note:** You can have more than one GitLab instance. 
 
-5. Add proxy config:
+5. Add the GitLab Filler Processor:
```
Comment (simonecorsi) [inline README.md:126]: I would briefly explain why we now need a BE plugin too

Comment (antoniomuso) [reply README.md:126]: Yes, sure. I want to write it in the migration guide. WDYT?

Comment (simonecorsi) [reply README.md:134]: I think you can do both, one for people migrating from old version and just a couple of words in the main readme just to specify why that is needed!

Comment (antoniomuso) [reply README.md:134]: ok

File: README.md [hunk 2]
```diff
@@ -121,32 +129,90 @@ integrations:
...
+    // Add this line
+    builder.addProcessor(new GitlabFillerProcessor(env.config));
+    //...
+    const { processingEngine, router } = await builder.build();
+    await processingEngine.start();
+    return router;
+}
+```
+
+This allows auto-filling of the annotations.
```
Comment (simonecorsi) [inline README.md:156]: Please add  "like the project id and slug" and it's good to me! Also, move this under `5. Add the GitLab Filler Processor:` so it's immediately clear what it does!

**Review by simonecorsi — CHANGES_REQUESTED** [review]
Just a small change and LGTM! 🙏

File: README.md (hunk 2 above)
Comment (antoniomuso) [reply README.md:156]: Done!

**Review by simonecorsi — APPROVED** [review]

**Pull request merged**